CodeEdit/NavigatorSidebar/NavigatorSidebar.swift

```diff
@@ -41,8 +43,22 @@ struct NavigatorSidebar: View {
                 .padding(.bottom, -8)
         }
         .safeAreaInset(edge: .bottom) {
-            NavigatorSidebarToolbarBottom(workspace: workspace)
-                .padding(.top, -8)
+            switch selection {
+            case 0:
+                NavigatorSidebarToolbarBottom(workspace: workspace)
+                    .padding(.top, -8)
```

Comment:
this -8 padding seems to be used multiple times, we should probably add a constant that holds it

Reply:
Maybe as a static let in `CodeEditUI`

```swift
public enum CodeEditUI {
  static let toolbarPadding: Double = -8.0
}
```

Reply:
Idk why it was added but the it looks the same even without the the padding

Reply:
It's only a problem once the content of the navigator grows larger than the height of the window. For testing purposes you can try removing this in the Project Navigator as you can see here:

<img width="260" alt="Screen Shot 2022-05-06 at 15 20 04" src="https://user-images.githubusercontent.com/9460130/167139450-886846e1-e7b6-4025-bd6d-840d46a73613.png">